release-pipeline.yml

```diff
@@ -1928,7 +1928,7 @@ stages:
                     if [[ $archetype == *"Arm"* ]]; then
                       echo "Skipping for ARM type on $distro"
                     else
-                      if [[ $distro == *"Mariner-2.0"* ]]; then
+                      if [[ $distro == *"Mariner-2.0"* || $distro == *"Mariner-3.0"* ]]; then
```

Comment:
We shall rather check for "Mariner-" instead of checking 2.0 and 3.0

Reply:
done